docs/studio-pro-plan.mdx

```diff
@@ -1,44 +1,47 @@
 ---
-title: Stately Studio Pro plan
+title: Stately Studio Pro Plan
 ---
 
-# Stately Studio Pro
+# Stately Studio Pro Plan
 
-[Upgrade to a Pro account](https://stately.ai/pricing) to get features designed to help you collaborate with your team, including the following Pro features.
+[Upgrade to a Pro account](https://stately.ai/pricing) to create more expressive flows, work faster with AI assistance, and achieve greater efficiency via editing and version control. A Pro account includes the following features:
```

Comment:
Do you think using `via editing` here makes it sound like editing machines is a Pro-only feature?

Reply:
Yes good point. I'll tweak that one!

Reply:
I changed it to be "Upgrade to a Pro account to create more expressive flows, work faster with AI assistance, and achieve greater flexibility in editing with version control."